### PR TITLE
Fix/et 1134 email from displays encoded html entities

### PR DESCRIPTION
**Pull request by redscar**
### 🎫 Ticket

[ET-1134] 

### 🗒️ Description

Single quotes were being escaped. Gmail did not like this. I removed the escaping that was occurring on the persons name

Bug Fix -

Single line change for TC sending emails for Paypal.

### 🎥 Artifacts 

https://user-images.githubusercontent.com/12241059/122456218-4e81f980-cf7b-11eb-86c5-d9eb7d700884.mp4

### ✔️ Checklist
- [x] I've included a changelog entry both in readme.txt and changelog.txt fi>
- [x] My code is tested. <!-- Check that tests are passing and DO NOT merge i>
- [x] My code has [proper inline documentation](https://the-events-calendar.g>

[ET-1134]: https://theeventscalendar.atlassian.net/browse/ET-1134

### CODE REVIEW

File: src/Tribe/Tickets.php [hunk 1]
```diff
@@ -2392,9 +2392,11 @@ public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
 
 				$headers[] = sprintf(
 					'From: %1$s <%2$s>',
-					filter_var( $from_name, FILTER_SANITIZE_STRING ),
+					stripcslashes($from_name),
```
Comment (juanfra) [inline src/Tribe/Tickets.php:2395]: We need to have empty spaces between the var when sending as a parameter
```suggestion
					stripcslashes( $from_name ),
```

File: src/Tribe/Tickets.php [hunk 2]
```diff
@@ -2392,9 +2392,11 @@ public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
...
 					$from_email
 				);
+				//html_entity_decode( $row[ $column_id ] , ENT_QUOTES | ENT_XML1, 'UTF-8');
+
```
Comment (juanfra) [inline src/Tribe/Tickets.php:2399]: Is this related to the other PR?

**Review by juanfra — CHANGES_REQUESTED** [review]
Nice fix! Looks like we need to use a different base branch for this PR (`G21.carnotaurus`). There is code that's unrelated to the PR.

**Comment by redscar** [comment]
Updated to new pull request https://github.com/the-events-calendar/event-tickets/pull/2023
